### ISSUE TITLE
First person camera and control scheme

### ISSUE THREAD

**Issue by c42f** [c42f/displaz#15]
For the FPS players out there who undoubtedly won't stop requesting this if it's not supported... and for the potential Oculus Rift support which will require something similar.

**Comment by nigels-com** [comment]
Do we know anything about where the ground is?  Which way is up?

**Comment by c42f** [comment]
Yes, z is usually up and I think assuming this will be fine for now.  For the cases where it's not, I currently include a trackball camera option (see View->Use trackball camera).

**Comment by landersson** [comment]
I'd love to see FPS style navigation added to displaz.
